src/backend/access/zedstore/zedstore_tidpage.c

```diff
@@ -1915,6 +1940,10 @@ zsbt_binsrch_tidpage(zstid key, Page page)
 	return low - 1;
 }
 
+/*
+ * It must be called after XLogBeginInsert and XLogEnsureRecordSpace have been
```

Comment:
Any assertion we can add to validate `XLogEnsureRecordSpace()` was called when we get to this function?

Reply:
Unfortunately, the globals that `XLogEnsureRecordSpace()` and `XLogInsertInit()` set are static and not exposed. Thus, we can't really assert that they have been called.